lavamoat/node/policy.json

```diff
@@ -1729,6 +1805,92 @@
         "source-map": true
       }
     },
+    "eslint": {
+      "builtin": {
+        "assert": true,
+        "fs.existsSync": true,
+        "fs.lstatSync": true,
+        "fs.readFileSync": true,
+        "fs.readdirSync": true,
+        "fs.statSync": true,
+        "fs.unlinkSync": true,
+        "fs.writeFile": true,
+        "fs.writeFileSync": true,
```

Comment:
disable in override

Reply:
experiment in making override, not compatible due to the value being partially consumed https://github.com/MetaMask/metamask-extension/pull/12091
perhaps lavamoat should invoke some sort of [fuckitjs](https://github.com/mattdiamond/fuckitjs) in order to violently rip out code that tries to consume the dis-allowed caps

Reply:
That's amazing.